inc/config.class.php

```diff
@@ -333,17 +333,35 @@ static function showConfigForm($item) {
       echo "</td></tr>";
 
       echo "<tr class='tab_bg_1'>";
-      echo "<td>".__('Category is mandatory when you assign a ticket', 'behaviors')."</td><td>";
-      Dropdown::showYesNo("is_ticketcategory_mandatory_on_assign",
-                          $config->fields['is_ticketcategory_mandatory_on_assign']);
-      echo "</td>";
-      echo "<th colspan=2' class='tab_bg_2 b center'>".__('New change')."</th></tr>";
+//      echo "<td>";
+      //GLPI 10 force category on update
+//      echo __('Category is mandatory when you assign a ticket', 'behaviors');
```

Comment:
i made many tests on existing ticket with out category defined (add a technician, change status...)
and category is still empty